.github/workflows/ci.yml

```diff
@@ -8,43 +8,98 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
+
+  test:
+
     runs-on: ubuntu-20.04
     continue-on-error: ${{ matrix.experimental }}
 
     strategy:
       matrix:
         include:
           - mediawiki_version: '1.35'
-            semantic_mediawiki_version: 4.0.2
+            smw_version: '4.1.0'
```

Comment:
let's stick with the latest SMW release if there's no reason to test against 4.1.0